database/dbHolidayMealBag.php

```diff
@@ -16,4 +16,32 @@ function get_data_by_family_id($id){
         return $row; //return the data as an associative array;
     }
 
+}
+
+function getHolidayMealBagSubmissions() {
+    $conn = connect();
+    $query = "SELECT * FROM dbHolidayMealBagForm JOIN dbFamily USING(id);";
```

Comment:
If I just search by Holiday Meal Bag, nothing shows up for me even though I do have data in dbHolidayMealBagForm. I think somewhere in this function might be a bug? It might have to do with this query on line 23